### PR TITLE
Always spawn a process for cli

### PR DESCRIPTION
**Pull request by smelukov**
**What kind of change does this PR introduce?**

feature
closes https://github.com/webpack/webpack-cli/issues/1084

**Did you add tests for your changes?**

no

**If relevant, did you update the documentation?**

no

**Summary**

Always spawn a process for cli

**Does this PR introduce a breaking change?**

no

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -4,10 +4,14 @@
```
Comment (ematipico) [inline bin/cli.js]: Is there a reason why you added this logic here instead of `bootstrap.js`?

Comment (smelukov) [reply bin/cli.js]: yep, bin/cli.js - is the entry point which includes logic of preparing data for webpack-cli and executes webpack-cli
boostrap - is the entry point of webpack-cli

Comment (rishabh3112) [reply bin/cli.js]: This change make sense to me,
`bootstrap.js` would contain logic to running CLI and this file will run CLI as a child process.

Comment (smelukov) [reply bin/cli.js]: why bin/cli.js can't exec boostrap.js?

Comment (smelukov) [reply bin/cli.js]: i think that bin/cli.js is a good place for it

Comment (rishabh3112) [reply bin/cli.js]: Is there any problem? I too think `bin/cli.js` is a good place.

Comment (ematipico) [reply bin/cli.js]: Just a lost in translation misunderstanding :)
Thank you for the explanation!

File: packages/generators/__tests__/init-generator.test.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { join } from 'path';
 import { run } from 'yeoman-test';
 
-describe('init generator', () => {
+// fixme: unstable
```
Comment (anshumanv) [inline packages/generators/__tests__/init-generator.test.ts]: Is this a flaky test? Seems to pass locally.

Comment (ematipico) [reply packages/generators/__tests__/init-generator.test.ts]: Locally but on Travis it times out

Comment (anshumanv) [reply packages/generators/__tests__/init-generator.test.ts]: Maybe we can increase the timeout for these tests?

Comment (ematipico) [reply packages/generators/__tests__/init-generator.test.ts]: Make a PR and let's see

Comment (anshumanv) [reply packages/generators/__tests__/init-generator.test.ts]: Sure thing 👍

**Pull request merged**